control-plane/roles/monitoring/templates/prometheus-scrape-config.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: prometheus-prometheus-scrape-confg
+data:
+  additional-scrape-configs.yaml: {{ prometheus_scrape_config | b64encode }}
```

Comment:
Where is this variable defined?

Reply:
https://github.com/metal-stack/metal-roles/commit/52584a27755c4fbc6c1982de4df4620e341ef853